Crypto-Currency/Crypto-Currency/Model/Networking/RepositoryAPI.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  RepositoryAPI.swift
+//  Crypto-Currency
+//
+//  Created by namtrinh on 06/08/2021.
+//
+
+import Foundation
+
+class RepositoryAPI {
+    func getCoins(urlString: String, completion: @escaping ([Coin]) -> () ) {
+        APIService.shared.request(urlString: urlString, expecting: Response<DataCoin>.self) { result in
+            switch result {
+            case .success(let coins):
+                completion(coins.data.coins)
+            case .failure(let error):
+                print(error)
+            }
+        }
+    }
+}
```

Comment:
Thêm phần headers chưa API key vào bạn. Nó sẽ fix toàn bộ lỗi data resopnse ở VC.